app/views/admin/assortments/_form.html.slim

```diff
@@ -1,10 +1,17 @@
-.row
-  = form_for @assortment, url: url, method: method, remote: true, builder: StandartBuilder do |f|
-    div= f.error_messages
-    .row
-      .col-md-2= f.label t('.name')
-      .col-md-5= f.text_field :name
-      .col-md-5= select_tag :category_id, options_from_collection_for_select(@categories, 'id', 'name')
-    .row
-      .col-md-6= f.submit t('save')
-      .col-md-6= link_to t('cancel'), 'javascript: void(false)', onclick: 'Popup.hideModal()'
+.modal-dialog
+  .modal-content
```

Comment:
I think it would be better to add .modal-dialog and .modal-content to shared/_modal/window partial. After this you will update only .modal-content